src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$hoverBlueColor: #00acdc;
+$hoverWhiteColor: #fff;
```

Comment:
You need to fix the variable names, they should show their purpose, not their content. if I decided to change the color, I would have to change it everywhere, not just in the variable definition.